workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -30,7 +31,48 @@ public class SpatialOSComponent : MonoBehaviour
         /// </summary>
         public WorkerSystem Worker;
 
+        /// <summary>
+        ///     An event that triggers when a worker flag changes.
+        /// </summary>
+        public event Action<string, string> OnWorkerFlagChange
+        {
+            add => flagSystem.OnWorkerFlagChange += value;
+            remove => flagSystem.OnWorkerFlagChange -= value;
```

Comment:
this should track what values are added/removed and when the monobehaviour is disabled to clean any remaining ones from the worker system

Reply:
Ew. Okay - this might need to be refactored slightly